### DISCUSSION TITLE
Datagrid edit without delete

### DISCUSSION THREAD

**Discussion by jnoneman** [Megabit/Blazorise#2217] [Q&A] [answered]
Is there a way to have "Edit" without "Delete"?  I have grid where only one column is editable, but I do not want to allow deleting of the row, It seems that if I put in a DataGridCommandColumn, I get both Edit and Delete.

**Comment by tomironkko** [comment]
You must use the Command Templates:

Something like this:

                <DataGridCommandColumn TItem="something">
                    
                    <NewCommandTemplate >
                    </NewCommandTemplate>
                    <EditCommandTemplate>
                        
                        <Button Color="Color.Primary" Outline="true" Clicked="@Clicked">Edit</Button>
                            
                    </EditCommandTemplate>
                    <DeleteCommandTemplate>
                        

                            
                    </DeleteCommandTemplate>
                    
                </DataGridCommandColumn>

**Comment by stsrki** [comment, marked as answer]
Or you can just disable the delete command

```cshtml
<DataGridCommandColumn TItem="Employee" DeleteCommandAllowed="false">
```
